### ISSUE TITLE
[BUG] Crash when clearing successful uploads

### ISSUE THREAD

**Issue by davigonz** [owncloud/android#2563]
### Actual behaviour
- App crashes when clearing successful uploads.

### Expected behaviour
- App should behave normally when clearing successful uploads.
 
### Steps to reproduce
1. Upload a file.
2. Wait till is completed.
2. Go to uploads view.
3. Tap three dots button and press "Clear successful".

### Environment data
Android version: 9

Device model: Google Pixel 2

Stock or customized system: Stock

ownCloud app version: 2.11 beta

ownCloud server version: 10.0.10

#### ownCloud log (data/owncloud.log)
```
2019-06-03 09:14:19.166 15125-15125/com.owncloud.android.debug E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.owncloud.android.debug, PID: 15125
    java.lang.IllegalArgumentException: Selection not allowed, use parameterized queries
        at com.owncloud.android.providers.FileContentProvider.delete(FileContentProvider.java:272)
        at com.owncloud.android.providers.FileContentProvider.delete(FileContentProvider.java:261)
        at android.content.ContentProvider$Transport.delete(ContentProvider.java:341)
        at android.content.ContentResolver.delete(ContentResolver.java:1684)
        at com.owncloud.android.datamodel.UploadsStorageManager.clearSuccessfulUploads(UploadsStorageManager.java:443)
        at com.owncloud.android.ui.activity.UploadListActivity.onOptionsItemSelected(UploadListActivity.java:212)
        at android.app.Activity.onMenuItemSelected(Activity.java:3543)
        at androidx.fragment.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:436)
        at androidx.appcompat.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:196)
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
        at androidx.appcompat.app.ToolbarActionBar$2.onMenuItemClick(ToolbarActionBar.java:64)
        at androidx.appcompat.widget.Toolbar$1.onMenuItemClick(Toolbar.java:204)
        at androidx.appcompat.widget.ActionMenuView$MenuBuilderCallback.onMenuItemSelected(ActionMenuView.java:781)
        at androidx.appcompat.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:840)
        at androidx.appcompat.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:158)
        at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:991)
        at androidx.appcompat.view.menu.MenuPopup.onItemClick(MenuPopup.java:128)
        at android.widget.AdapterView.performItemClick(AdapterView.java:318)
        at android.widget.AbsListView.performItemClick(AbsListView.java:1159)
        at android.widget.AbsListView$PerformClick.run(AbsListView.java:3136)
        at android.widget.AbsListView$3.run(AbsListView.java:4052)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6680)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)

```

### PR

App: https://github.com/owncloud/android/pull/2566

**Comment by sd1998** [comment]
@davigonz @jesmrec I would like to work on this.

**Comment by davigonz** [comment]
> I would like to work on this.

Great, go for it

**Comment by jesmrec** [comment]
Fixed. To be released in 2.11
